### PR TITLE
ffmpeg: AV1 integration

### PR DESCRIPTION
**Pull request by bradh**
Obsoletes #1177

### CODE REVIEW

**Comment by bradh** [comment]
@n-kai-cj Can you check this works for your use case?

**Comment by n-kai-cj** [comment]
@bradh Thank you! This works.

File: ffmpeg/cppbuild.sh [hunk 1]
```diff
@@ -43,6 +45,10 @@ XML2=libxml2-2.9.12
 LIBSRT_VERSION=1.5.0
 WEBP_VERSION=1.2.4
 FFMPEG_VERSION=5.1.2
+LIBSRT_VERSION=1.4.4
+WEBP_VERSION=1.2.2
```
Comment (saudet) [inline ffmpeg/cppbuild.sh:49]: Do we need to use old versions of libsrt and webp to make this work?

Comment (bradh) [reply ffmpeg/cppbuild.sh:49]: I think just bad merge on my part. Will try cleaning that up.

Comment (bradh) [reply ffmpeg/cppbuild.sh]: Looks like its still OK.

**Pull request merged**